src/hb-coretext-shape.cc

```diff
@@ -160,30 +160,42 @@ release_data (void *info, const void *data, size_t size)
   hb_blob_destroy ((hb_blob_t *) info);
 }
 
+static CGFontRef
+create_cg_font (hb_blob_t *blob, unsigned int index)
+{
+  if (unlikely (index != 0))
+    return nullptr; // How?
```

Comment:
There is [`CTFontManagerCreateFontDescriptorsFromData()`](https://developer.apple.com/documentation/coretext/ctfontmanagercreatefontdescriptorsfromdata(_:)?language=objc), then propably use [`CTFontCopyGraphicsFont()`](https://developer.apple.com/documentation/coretext/ctfontcopygraphicsfont(_:_:)?language=objc) to get `CGFontRef` from the `CTFontDescriptorRef`.

Reply:
Ah that one. Yeah. It's slow since it loads all the faces at once :(. I'll see if I can use that.